src/write_gcm.c

```diff
@@ -2867,29 +2867,17 @@ static void wg_json_RFC3339Timestamp(json_ctx_t *jc, cdtime_t time_stamp);
 //   string collectd_version = 3;
 //   repeated CollectdPayload collectd_payloads = 4;
 // }
-static int wg_json_CreateCollectdTimeseriesRequest(_Bool pretty,
-    const monitored_resource_t *monitored_resource,
-    const wg_payload_t *head, const wg_payload_t **new_head,
-    char **json) {
```

Comment:
Given that this is in the 5.5.2 branch, can we please keep the original formatting? Also below.

Reply:
Done.